packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx

```diff
@@ -65,7 +65,7 @@ export const SettingNavigation = ({
           borderStyle: 'solid',
           borderWidth: 0,
           borderBottomWidth: 1,
```

Comment:
Is there any reason for keeping the border here? instead of removing all the border related css?